pkg/kubelet/pluginmanager/cache/actual_state_of_world.go

```diff
@@ -53,6 +53,9 @@ type ActualStateOfWorld interface {
 	// If a plugin does not exist with the given socket path, this is a no-op.
 	RemovePlugin(socketPath string)
 
+	// GetPlugin retrieves PluginInfo by socketPath
+	GetPlugin(socketPath string) PluginInfo
```

Comment:
This should no longer be needed, please remove.

Reply:
This is still used by operation_generator 
```
		pluginInfo := actualStateOfWorldUpdater.GetPlugin(socketPath)
```

Reply:
https://github.com/kubernetes/kubernetes/pull/89934#discussion_r406971435